### DISCUSSION TITLE
Warp all/nearby party members

### DISCUSSION THREAD

**Discussion by da-rulez** [AscensionGameDev/Intersect-Engine#1516] [Ideas]
**Is your feature request related to a problem? Please describe.**

Now you need to fiddle with events and variables to warp a party. 

**Describe the solution you'd like**

It would be nice if there was an extra option to the warp function (in event and/or map editor) that would warp all party members with you.

**Comment by Arufonsu** [comment]
sounds useful, but this is probably getting worked out along with coming features such as instances, where party play might be the main focus.

**Comment by da-rulez** [comment]
Until then I'll keep it up as a suggestion 😉

**Comment by Cheshire92** [comment]
I'd like to add a second tickbox to this if we do this that only warps nearby party members.

So the developer gets the option to not just warp everyone from across the world every time a user does something. Lol

**Comment by da-rulez** [comment]
Makes sense!

I figured there'd be a limit on how far away a party member could be! Maybe an option to cancel the warp if not all party members are close enough?
